Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.7-alpine
+
+WORKDIR /px-ldap-replace
+
+COPY * ./
```

A asterisk is not needed here.
```suggestion
COPY . .
```
But even better would be to only copying what you really need.